federation-js/src/composition/__tests__/composeAndValidate.test.ts

```diff
@@ -921,6 +921,45 @@ describe('composition of schemas with directives', () => {
       );
     }
   });
+
+  it('should merge @tag on Object Type fields', () => {
```

Comment:
I'm not sure I follow what this is testing — if it's about merging tag, shouldn't there be tags on both of them? Or is the point that moving `@tag` over was just failing overall? I tried and it looks like the test passes without the code change.

Reply:
Yep moving @tag was failing overall, but the conditions were pretty specific (i.e. the `@external` has to exist so to trigger the right branch in merging logic)